src/main.js

```diff
@@ -1,9 +1,14 @@
 #!/usr/bin/env node
+if (process.versions.node.split('.')[0] < 5) {
+  throw new Error('Only Node 5+ is supported');
+}
+require('source-map-support').install();
```

Comment:
I kept running it accidentally locally with node 4 and it broke in a *really* obscure way, even after i deleted node_modules and redid npm install....  normally I wouldn't do this, but this should be silent for anything that works.  I'll remove this if you still think I should :)